src/clims/services/extensible.py

```diff
@@ -41,12 +41,16 @@ def get_implementation(self, extensible_type_full_name):
 
     def get_extensible_type(self, org, name):
         """
-        Returns an extensible_type by name. Can be cached.
+        Returns an extensible_type by name. May be from a cache.
```

Comment:
May be -> Maybe?

Reply:
I intended to write "may be" as "it's allowed to be" rather than "perhaps". Both forms exist. I think I'll just remove this as that should be hidden from the caller anyways.